### PR TITLE
fix: refine example grafana dashboard

### PR DESCRIPTION
**Pull request by Yiyiyimu**
### What this PR does / why we need it:
Ref #3402

- update grafana to 'for external use' version
- add screenshot of new panels

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: doc/json/apisix-grafana-dashboard.json
Comment (Firstsawyou) [inline doc/json/apisix-grafana-dashboard.json:1520]: Missing a newline at end of file.

Comment (Yiyiyimu) [reply doc/json/apisix-grafana-dashboard.json]: Thanks! Fixed!

**Review by membphis — CHANGES_REQUESTED** [review]

**Comment by tokers** [comment]
I think the HTTP requests latency can be updated to only show the 95th percentile (or 90th?), it's not easy to understand now.

File: doc/plugins/prometheus.md
Comment (Yiyiyimu) [reply doc/plugins/prometheus.md:123]: There are currently two problems for histogram:
1. The legends have leading for trailing zeros ([reason](https://github.com/knyar/nginx-lua-prometheus/blob/499de49cab0217c855dd9be12c3c1596cc2e9827/prometheus.lua#L24-L27)) so it looks messy
2. There are two many buckets out there. We don't need buckets like "30,40,50,60,70,80,90,100,200,300,400,500". After reduce buckets, it somewhat looks better
image

Or we could use 95-percentile as @tokers said. I'm not so familiar with monitor APISIX, so what do you think about that

Comment (Yiyiyimu) [reply doc/plugins/prometheus.md:123]: For `etcd-reachable` and `nginx metric errors`, the comments of what these two metrics do are explained on the metrics page alongside with the data. And for how I implement it
- `etcd-reachable`: when etcd is reachable, the metric would be 1 and the graph would be green. While it is disconnected, the metric turns to 0 and the panel would be red.
- `nginx metric errors`: it measures how many errors of nginx-lua-prometheus. I also leave it to only show the current error number without time series, since if it is larger than 0, it should be taken care of. And also if it is larger than 0, it would turn red.

Comment (Yiyiyimu) [reply doc/plugins/prometheus.md:123]: would fix leading for trailing zeros in https://github.com/knyar/nginx-lua-prometheus/pull/119

**Comment by Yiyiyimu** [comment]
waiting for knyar/nginx-lua-prometheus#119 got merged

File: doc/plugins/prometheus.md
Comment (Yiyiyimu) [reply doc/plugins/prometheus.md:123]: Hi @membphis PTAL

Updated to newer version of nginx-lua-prometheus, and it would looks like
image

**Comment by membphis** [comment]
> I think the HTTP requests latency can be updated to only show the 95th percentile (or 90th?), it's not easy to understand now.

agree +1

**Review by membphis — APPROVED** [review]
LGTM, and I think we can update this part too.

image

**Review by spacewander — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
> LGTM, and I think we can update this part too.
> 
> image

Hi @LiteSun could you offer some help here? I'm not familiar with frontend layout 😬

**Pull request merged**
